### DISCUSSION TITLE
Server process disappearing

### DISCUSSION THREAD

**Discussion by s1037989** [mojolicious/mojo#2211] [Q&A]
Does anyone have any experience with morbo/daemon/prefork processes just dying?  I find this most common when they are running for a while, especially in a tmux or vscode ssh session and the shell goes away too!  Moreover, I find this most common when the system is running very low on RAM.  But why is the process just dying without any errors or warnings, how is it taking out the shell, and why -- oh, sorry, *I was going to say why isn't the process rolling into swap?  This system has no swap configured.  **Is that the explanation?***

**Comment by Grinnz** [comment]
oomkiller will kill processes using the most memory when it fills up, and not having swap means there is no other recourse, so probably yes

**Comment by s1037989** [comment]
Thank you for the confirmation! I'll enable swap and if I notice processes
dying less I'll report back to close the loop.
